src/styles/nav.scss

```diff
@@ -0,0 +1,42 @@
+.nav {
+  text-transform: uppercase;
+  font-weight: 500;
+
+  &__list {
+    display: flex;
+    list-style: none;
+
+    & :not(:last-child) {
+      margin-right: 20px;
+    }
+  }
+
+  &__link {
+    position: relative;
+    display: block;
+    text-decoration: none;
+    color: $main-accent;
+    padding-block: 23px;
+    transition: $transition-duration;
```

Comment:
Ensure that the SCSS variable `$transition-duration` is defined in your stylesheets. If it's not defined, it will cause an error during the SCSS compilation.